crates/iota-move/src/main.rs

```diff
@@ -28,10 +28,10 @@ struct Args {
     pub package_path: Option<PathBuf>,
     /// If true, run the Move bytecode verifier on the bytecode from a
     /// successful build
-    #[arg(long = "path", short = 'p', global = true)]
```

Comment:
This is a breaking change so the PR should be flagged as such

Reply:
small update: it should panic, but it doesn't. A small test program I wrote noticed duplicate long names and correctly panicked. But when I run the current `iota-move` binary on develop it shows this:
```
Options:
  -p, --path <PACKAGE_PATH>
          Path to a package which the command should be run with respect to
  -p, --path
          If true, run the Move bytecode verifier on the bytecode from a successful build
  -p, --path
          If true, print build diagnostics to stderr--no printing if false
 ```
 instead of panicking. :thinking:

Reply:
just tested clap's behavior. If I call `iota-move --path true build` it complains with: `No such file or directory (os error 2)`, which means it picks the option that comes first. So technically this is not a breaking change, because no one could have ever used the other options with the long parameter.